lib/seahorn/BvOpSem2.cc

```diff
@@ -643,6 +643,7 @@ class OpSemVisitor : public InstVisitor<OpSemVisitor>, OpSemVisitorBase {
     if (!m_ctx.getMemReadRegister()) {
       LOG("opsem", ERR << "No read register found - check if corresponding"
                           "shadow instruction is present.");
+      m_ctx.setMemReadRegister(Expr());
```

Comment:
seems dangerous. This is unexpected. The program should crash or exit at this point.

Reply:
I am copying the pattern here: https://sourcegraph.com/github.com/seahorn/seahorn@dev10/-/blob/lib/seahorn/BvOpSem2.cc#L1793

Does the original pattern also have the same problem?